src/StaticWebAssetsSdk/Tasks/Data/ContentTypeProvider.cs

```diff
@@ -0,0 +1,444 @@
+            [".inf"] = new ContentTypeMapping("application/octet-stream", null, "*.inf", 1),
+            [".java"] = new ContentTypeMapping("application/octet-stream", null, "*.java", 1),
+            [".jpb"] = new ContentTypeMapping("application/octet-stream", null, "*.jpb", 1),
+            [".lpk"] = new ContentTypeMapping("application/octet-stream", null, "*.lpk", 1),
+            [".lzh"] = new ContentTypeMapping("application/octet-stream", null, "*.lzh", 1),
+            [".mdp"] = new ContentTypeMapping("application/octet-stream", null, "*.mdp", 1),
+            [".mix"] = new ContentTypeMapping("application/octet-stream", null, "*.mix", 1),
+            [".msi"] = new ContentTypeMapping("application/octet-stream", null, "*.msi", 1),
+            [".mso"] = new ContentTypeMapping("application/octet-stream", null, "*.mso", 1),
+            [".ocx"] = new ContentTypeMapping("application/octet-stream", null, "*.ocx", 1),
+            [".pcx"] = new ContentTypeMapping("application/octet-stream", null, "*.pcx", 1),
+            [".pcz"] = new ContentTypeMapping("application/octet-stream", null, "*.pcz", 1),
+            [".pfb"] = new ContentTypeMapping("application/octet-stream", null, "*.pfb", 1),
+            [".pfm"] = new ContentTypeMapping("application/octet-stream", null, "*.pfm", 1),
+            [".prm"] = new ContentTypeMapping("application/octet-stream", null, "*.prm", 1),
+            [".prx"] = new ContentTypeMapping("application/octet-stream", null, "*.prx", 1),
+            [".psd"] = new ContentTypeMapping("application/octet-stream", null, "*.psd", 1),
+            [".psm"] = new ContentTypeMapping("application/octet-stream", null, "*.psm", 1),
+            [".psp"] = new ContentTypeMapping("application/octet-stream", null, "*.psp", 1),
+            [".qxd"] = new ContentTypeMapping("application/octet-stream", null, "*.qxd", 1),
+            [".rar"] = new ContentTypeMapping("application/octet-stream", null, "*.rar", 1),
+            [".sea"] = new ContentTypeMapping("application/octet-stream", null, "*.sea", 1),
+            [".smi"] = new ContentTypeMapping("application/octet-stream", null, "*.smi", 1),
+            [".snp"] = new ContentTypeMapping("application/octet-stream", null, "*.snp", 1),
+            [".thn"] = new ContentTypeMapping("application/octet-stream", null, "*.thn", 1),
+            [".toc"] = new ContentTypeMapping("application/octet-stream", null, "*.toc", 1),
+            [".u32"] = new ContentTypeMapping("application/octet-stream", null, "*.u32", 1),
+            [".xsn"] = new ContentTypeMapping("application/octet-stream", null, "*.xsn", 1),
+            [".xtp"] = new ContentTypeMapping("application/octet-stream", null, "*.xtp", 1),
+        };
+
+    internal ContentTypeMapping ResolveContentTypeMapping(string relativePath, TaskLoggingHelper log)
+    {
+        var builtIn = ResolveBuiltIn(relativePath, log);
```

Comment:
It looks like `ResolveBuiltIn()` gets called regardless of whether a custom mapping is resolved. Is that necessary?